pkg/features/features.go

```diff
@@ -106,7 +107,7 @@ var allFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	ManagedIstio:       {Default: true, PreRelease: featuregate.Deprecated},
 	APIServerSNI:       {Default: true, PreRelease: featuregate.Deprecated},
 	SeedChange:         {Default: true, PreRelease: featuregate.Beta},
-	ReversedVPN:        {Default: true, PreRelease: featuregate.Beta},
+	ReversedVPN:        {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Comment:
Can you delete this constant now? https://github.com/gardener/gardener/blob/b53d99f3edadcfa6d1fb66510debbf5777a92209/pkg/apis/core/v1beta1/constants/types_constants.go#L560-L561

Reply:
done